tests/test_androidtv_sync.py

```diff
@@ -123,6 +123,33 @@ def test_start_intent(self):
                 getattr(self.atv._adb, self.ADB_ATTR).shell_cmd, "am start -a android.intent.action.VIEW -d TEST"
             )
 
+    def test_send_intent(self):
```

Comment:
Can you please remove `test_send_intent`. The underlying source code is already tested elsewhere.